packages/geolocator/tizen/src/geolocator_tizen_plugin.cc

```diff
@@ -90,111 +90,101 @@ class GeolocatorTizenPlugin : public flutter::Plugin {
     geolocator_updates_channel_->SetStreamHandler(std::move(handler));
```

Comment:
Please consider extending the `flutter::StreamHandler` (`FlStreamHandler`) class to create a stream handler class as I've done for [battery_plus_tizen](https://github.com/flutter-tizen/plugins/pull/351) and [connectivity_plus_tizen](https://github.com/flutter-tizen/plugins/pull/352).

Reply:
I haven't applied the StreamHandler class yet. Because I still don't perfect understand both StreamHandler and LocationManager. If that's okay with you, I'd like to make this a separate PR.

Reply:
In order to refactor the `LocationManager` class, you may refer to

- `Connection::StartListen()` of connectivity_plus_tizen which returns false on immediate error and takes a result callback which returns a `ConnectionType` on completion (Note that the `ConnectionType` enum has a `kError` value to represent an error state.)
- `ServiceManager` of permission_handler_tizen whose return type is `ServiceStatus`

Reply:
I extended 'LocationStream' and 'ServiceStatusStream' from 'LocationManager' to StreamHandler.
And I changed Error handling of `LocationManager` from `TizenResult` to throw exception. (For minimize use of output parameters)